gammapy/image/core.py

```diff
@@ -898,3 +998,17 @@ def __str__(self):
             info += self[name].__str__()
             info += '\n'
         return info
+
+
+def _get_resampled_wcs(skyimage, factor, downsampled):
```

Comment:
Only pass `wcs` in here, not `skyimage`.

I think fermipy has a slightly more flexible wcs utils function for this ... I'll start comparing and merging the two tomorrow.